database/migration/migrateData.ts

```diff
@@ -1060,75 +1060,285 @@ const migrateData = async () => {
     await DB.changeSchema(
-      `ALTER TYPE permission_policy_type ADD VALUE IF NOT EXISTS 'VIEW' after 'ASSIGN'`
+      `ALTER TABLE application_list_shape
+        DROP COLUMN IF EXISTS total_questions;
+      ALTER TABLE application_list_shape
+        DROP COLUMN IF EXISTS total_assigned_submitted;
+      ALTER TABLE application_list_shape
+        DROP COLUMN IF EXISTS total_assigned;
+      ALTER TABLE application_list_shape
+        DROP COLUMN IF EXISTS total_assign_locked;`
     )
-  }
 
-  console.log(' - Remove uniqueness constraint from data_view table')
-  await DB.changeSchema(`
-    ALTER TABLE data_view DROP CONSTRAINT IF EXISTS outcome_display_table_name_code_key; 
-  `)
+    await DB.changeSchema(
+      `CREATE OR REPLACE FUNCTION application_list (userid int DEFAULT 0)
+        RETURNS SETOF application_list_shape
+        AS $$
+        SELECT
+            app.id,
+            app.serial,
+            app.name,
+            template.code AS template_code,
+            template.name AS template_name,
+            CONCAT(first_name, ' ', last_name) AS applicant,
+            org.name AS org_name,
+            stage_status.stage,
+            stage_status.stage_colour,
+            stage_status.status,
+            app.outcome,
+            status_history_time_created AS last_active_date,
+            ts.time_scheduled AS applicant_deadline,
+            assigners,
+            reviewers,
+            reviewer_action,
+            assigner_action
+        FROM
+            application app
+        LEFT JOIN TEMPLATE ON app.template_id = template.id
+        LEFT JOIN "user" ON user_id = "user".id
+        LEFT JOIN application_stage_status_latest AS stage_status ON app.id = stage_status.application_id
+        LEFT JOIN organisation org ON app.org_id = org.id
+        LEFT JOIN assignment_list (stage_status.stage_id) ON app.id = assignment_list.application_id
+        LEFT JOIN review_list (stage_status.stage_id, $1, stage_status.status) ON app.id = review_list.application_id
+        LEFT JOIN assigner_list (stage_status.stage_id, $1) ON app.id = assigner_list.application_id
+        LEFT JOIN trigger_schedule ts ON app.id = ts.application_id
+            AND ts.is_active = TRUE
+            AND ts.event_code = 'applicantDeadline'
+    WHERE
+        app.is_config = FALSE
+    $$
+    LANGUAGE sql
+    STABLE;`
+    )
+
+    console.log(' - Convert default for template_element.is_reviewable to ONLY_IF_APPLICANT_ANSWER')
+
+    await DB.changeSchema(`
+    ALTER TYPE public.is_reviewable_status ADD VALUE IF NOT EXISTS
+    'ONLY_IF_APPLICANT_ANSWER' AFTER  'NEVER';`)
+
+    await DB.changeSchema(`
+      ALTER TABLE public.template_element 
+        ALTER COLUMN is_reviewable
+        SET DEFAULT 'ONLY_IF_APPLICANT_ANSWER';`)
+
+    await DB.changeSchema(`
+    UPDATE template_element te SET is_reviewable = 'ONLY_IF_APPLICANT_ANSWER'
+      WHERE te.is_reviewable IS NULL
+        AND te.category = 'QUESTION'`)
+  }
```

Comment:
You'll also need to add the NOT NULL constraints, and if the field is NON NULLABLE, then you'll have to remove the `AND te.category = 'QUESTION'` bit from above, I think. 

(I don't think there's much point trying to preserve NULL for non-question elements, as they'll all get the default value when created anyway)